### PR TITLE
refactor: update project tags when editor page has mounted

### PR DESCRIPTION
**Pull request by kunzms**

### CODE REVIEW

File: src/services/projectService.ts [hunk 1]
```diff
@@ -170,20 +170,31 @@ export default class ProjectService implements IProjectService {
...
+    private async getTagsFromPreExistingLabelFiles(
+        project: IProject,
+        storageProvider: IStorageProvider,
+        asset?: string) {
         const tags: ITag[] = [];
         const tagNameSet = new Set<string>();
         try {
             const blobs = new Set<string>(await storageProvider.listFiles());
+            const assetLabel = asset ? asset + constants.labelFileExtension : undefined;
+            console.log(assetLabel);
```
Comment (stew-ro) [inline src/services/projectService.ts:184]: console log

File: src/react/components/pages/editorPage/editorPage.tsx [hunk 2]
```diff
@@ -534,7 +534,7 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
         }
 
         const assetMetadata = await this.props.actions.loadAssetMetadata(this.props.project, asset);
-        await this.props.actions.updateProjectTagsFromFiles(this.props.project);
+        await this.props.actions.updateProjectTagsFromFiles(this.props.project, asset.name);
```
Comment (stew-ro) [inline src/react/components/pages/editorPage/editorPage.tsx:537]: Now takes 500-3000 milliseconds to perform on my laptop for one asset. Better than the 7000 milliseconds it took before. 

Could just call once when editor page mounts for all assets, not per asset on asset select.

File: src/react/components/pages/editorPage/canvas.tsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ import Polygon from "ol/geom/Polygon";
 import HtmlFileReader from "../../../../common/htmlFileReader";
 import { parseTiffData, renderTiffToCanvas, loadImageToCanvas } from "../../../../common/utils";
 import { constants } from "../../../../common/constants";
-import { CanvasCommandBar } from "./CanvasCommandBar";
+import { CanvasCommandBar } from "./canvasCommandBar";
```
Comment (stew-ro) [inline src/react/components/pages/editorPage/canvas.tsx:35]: I get file not found now

Comment (kunzms) [reply src/react/components/pages/editorPage/canvas.tsx:35]: Windows is case insensitive, so I have to make the change in github online editor. Try sync again. If not work, rename the file manually.

**Pull request merged**